docs/src/app/components/Card/ExampleWithAvatar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import mui from 'material-ui';
```

Comment:
use direct imports so others can find them easier when they want to import specific components only. 

I mean like `import Avatar from 'material-ui/lib/avatar';`, etc.